cpix/filters.py

```diff
@@ -44,9 +44,31 @@ def parse(xml):
 class LabelFilter(CPIXComparableBase):
     """
     LabelFilter element
-    Not yet implemented
+    Has single required attribute:
+        label
     """
 
+    def __init__(self, label):
+        self.label = label
+
+    def element(self):
+        """Returns XML element"""
+        el = etree.Element("LabelFilter")
+        el.set("label", str(self.label))
+        return el
+
+    @staticmethod
+    def parse(xml):
+        """
+        Parse XML and return KeyPeriodFilter
```

Comment:
LabelFilter*

Reply:
Fixed in https://github.com/unifiedstreaming/pycpix/pull/8/commits/696916a07f329cb51dd9262417b13506649de1a3